.env.development

```diff
@@ -8,7 +8,7 @@
 # Locale for translation. Using a locale other than `en` tests the
 # successful fallback to `en`. You will also see up-to-date text used
 # in production
-LOCALE="en_AU"
+LOCALE="en_TEST"
```

Comment:
Part of the comment is not valid anymore:

> You will also see up-to-date text used in production

Reply:
True. The downside to this change is that we're not looking at a production locale. 
The upside is that it's agnostic of production locales.
🤷